rocAL/rocAL/source/augmentations/color_augmentations/node_exposure.cpp

```diff
@@ -26,35 +26,28 @@ THE SOFTWARE.
 
 ExposureNode::ExposureNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs),
-        _shift(SHIFT_RANGE[0], SHIFT_RANGE[1])
-{
-}
+        _exposure_factor(EXPOSURE_FACTOR_RANGE[0], EXPOSURE_FACTOR_RANGE[1]) {}
 
-void ExposureNode::create_node()
-{
+void ExposureNode::create_node() {
     if(_node)
         return;
 
-    _shift.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
-    // _node = vxExtrppNode_ExposurebatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _shift.default_array(), _batch_size);
+    _exposure_factor.create_array(_graph, VX_TYPE_FLOAT32, _batch_size);
+    _node = vxExtRppExposure(_graph->get(), _inputs[0]->handle(), _src_tensor_roi, _outputs[0]->handle(), _exposure_factor.default_array(), _input_layout, _output_layout, _roi_type);
```

Comment:
same comment as before for layout. Applies to all such

Reply:
Done